[ALLUXIO-1764] Remove unused constant alluxio.AlluxioURI#WINDOWS

yuanqingyecmu opened the pull request.

AmplabJenkins commented:
Can one of the admins verify this patch?

yupeng9 commented:
okay to test

yupeng9 commented on core/common/src/main/java/alluxio/AlluxioURI.java:
```diff
@@ -37,7 +37,12 @@
+   * For the task.
+   * The WINDOWS boolean variable is unused, so I comment this line of code.
+   */
+//  private static final boolean WINDOWS = OSUtils.isWindows();
```
This variable can be removed, and therefore not necessary to be commented out.

apc999 commented:
Jenkins, test this please

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/Alluxio-Pull-Request-Builder/8932/
Test PASSed.

yupeng9 commented:
LGTM. Thanks for the improvement.

yuanqingyecmu commented:
Thank you all!
Note that in the later commit, I removed the unused import
alluxio.util.OSUtils  to pass the style check, since it shows error after I
removed the unused WINDOW variable. Hope that I didn't mess things up.

Best,
Yuanqing

On Thu, Mar 17, 2016 at 1:15 PM, Yupeng Fu <notifications@github.com> wrote:

> Merged #2852 <https://github.com/Alluxio/alluxio/pull/2852>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Alluxio/alluxio/pull/2852#event-593745769>
>

yupeng9 commented:
No worries. Everything built.

yuanqingyecmu commented:
OK. Thanks!

On Fri, Mar 18, 2016 at 12:25 AM, Yupeng Fu <notifications@github.com>
wrote:

> No worries. Everything built.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Alluxio/alluxio/pull/2852#issuecomment-198202238>
>
